03-components.Rmd

```diff
@@ -1,10 +1,129 @@
 ```
+
+## Components of reproducibility
+
+A reproducible analysis is transparent, consistent, and accessible.
+
+- **Transparency** refers to the idea that it is well communicated and everything is displayed: data, code, goals, methods, and decisions. There are no secrets in a reproducible analysis/
+- **Consistency** refers to the idea that the code can be consistently run, but also everything follows a particular system, conventions and design.
+- **Accessibility** refers to the idea that anyone anywhere should be able to run and/or examine the analysis. No pay walls or expensive software should be required.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21b8e34c516_0_5")
+```
+
+## Transparent
+
+Code that runs reliably is important but clear communication about the analysis and the decisions made make an analysis reproducible. A transparent analysis is not only well communicated, but also shared publicly in a way that others can comment and contribute ideas and suggestions to it or borrow the methods and strategies for their own analyses.
+
+### Open source
+
+Open source means not making code and data publicly available but that the analysis is able to be modified by anyone. This doesn't mean that any and all modifying contributions need to be accepted, because some level of standards and quality checks need to be maintained by the owners of the analysis, but just that anyone online could propose a contribution if they wanted to.
+
+For an analysis to be truly open source, it needs to be easily accessed by others and stored online. Code that can be emailed, for example, is not considered open source.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Reproducible parrot is reading a journal article with data and code they are interested in. The journal article says ‘Code and data are available upon request by email’. The parrot sends an email that says ‘ The email is going to an inbox with 999,999,565473 emails in it and it is labeled ‘the corresponding author’s inbox’."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2006b5a2e6b_0_448")
+```
+
+For reproducibility, keeping your code on GitHub is a great open source solution. It is commonly used, and has a built in system that allows others to contribute changes in a way that can be methodically reviewed by you (this is called the pull request system and we will talk about it more).
+
+### Data is publicly available
+
+A transparent analysis has data that is publicly shared so that others can re-run the analysis as you have. Data should be provided in a way that it can be programmatically accessed (downloaded by a script). Data also need to be well documented in the form of metadata.
```

Comment:
```suggestion
A transparent analysis has data that is publicly shared so that others can re-run the analysis as you have. Data should be provided in a way that it can be programmatically accessed (downloaded by a script). Data also need to be well-documented in the form of metadata.
````

Reply:
I think either way works, but I recall the learning objectives using a hyphen

Reply:
Consistency is important (as I literally wrote in this chapter lol).